### PR TITLE
Update of article-management/ and its sub pages

### PR DESCRIPTION
**Pull request by fiedsch**
* updated the .de.md files too (wehere I thought they were wrong)
* insert-tags.en.md was assigned to @toflar in No. 35 so no update here

### CODE REVIEW

File: docs/manual/article-management/articles.en.md [hunk 1]
```diff
@@ -1,60 +1,57 @@
...
 ---
 
-{{% notice warning %}}
-This article is machine translated.
-{{% /notice %}}
-
 Article management is a separate module in Contao that you can *find* in the navigation at the top of the *Content* group. Each article is assigned to a specific page and layout area. In contrast to too many other CMSs, the inclusion of articles in Contao is not limited to the main column, so you can design your website flexibly.
 
-![The article management](/de/article-management/images/de/die-artikelverwaltung.png?classes=shadow)
+![The article management](/de/article-management/images/de/articlemanagement.png?classes=shadow)
```
Comment (netzarbeiter) [inline docs/manual/article-management/articles.en.md]:
```suggestion
![The article management](/de/article-management/images/en/articlemanagement.png?classes=shadow)
```
Bitte Bild in Ordner en verschieben

File: docs/manual/article-management/articles.en.md [hunk 2]
```diff
@@ -1,60 +1,57 @@
...
 
-Each page can contain any number of articles, which are displayed one below the other within their layout area in the order determined by you. Contao automatically detects whether the whole article or only the text followed by a `Weiterlesen`link should be displayed.
+Each page can contain any number of articles, which are displayed one below the other within their layout area in the order determined by you. Contao automatically detects whether the whole article or only the text followed by a `Read more` link should be displayed.
 
-![Several articles with teaser text](/de/article-management/images/de/mehrere-artikel-mit-teasertext.png?classes=shadow)
+![Several articles with teaser text](/de/article-management/images/de/multiple-articles-with-teasertext.png?classes=shadow)
```
Comment (netzarbeiter) [inline docs/manual/article-management/articles.en.md]:
```suggestion
![Several articles with teaser text](/de/article-management/images/en/multiple-articles-with-teasertext.png?classes=shadow)
```
Bitte Bild in Ordner en verschieben

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Andreas

**Comment by fiedsch** [comment]
OK. I'll move the images and adapt the paths. Will have time for that (probably) tonight.

**Comment by fiedsch** [comment]
Thanks for cross checking, Bjarke. Here you go.

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**